app/Model/Behavior/ProvisionerBehavior.php

```diff
@@ -1185,7 +1211,7 @@ protected function provisionGroups($model, $gmodel, $coGroupIds, $created, $prov
    * @throws InvalidArgumentException
    */
   
-  protected function provisionPeople($model, $pmodel, $coPersonIds, $created, $provisioningAction) {
+  protected function provisionPeople($model, $pmodel, $coPersonIds, $created, $provisioningAction, $originChange=array()) {
```

Comment:
update the phpDoc and maybe the description of the function

Reply:
Done